.eslintrc.js

```diff
@@ -20,6 +21,7 @@ module.exports = {
         "react"
     ],
     "rules": {
+        "max-len": [2, {"code": 120}]
```

Comment:
What is the 2 in this? just curious

Reply:
Indicates that eslint will show an error if this rule is broken.